requirements_test.txt

```diff
@@ -6,6 +6,7 @@
 -r requirements_test_pre_commit.txt
 codecov==2.1.10
 coverage==5.5
+freezegun==1.1.0
```

Comment:
What's freezegun adding, and how are you using it? Where is time impacting your code? We've done 7.5 years without

Reply:
It's used in the tests for the camera platform to freeze the time to match the timestamps of the json files when testing event handling. But I double checked and I seem to have worked around that. I'll remove it from this and all related PRs.

Reply:
Actually, I am using it actively in the tests for the sensors platform. I understand that you don't like adding dependencies for just a single integrations tests. I'll see how to do that without freezegun. It proved to be handy for pyatmo.